src/Dto/Request/CurrentSimPackageRequest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+declare(strict_types=1);
+
+namespace eSIM\eSIMCoreClient\Dto\Request;
+class CurrentSimPackageRequest extends BaseRequest
```

Comment:
SDK olduğu için ince eleyeyim su namespace'den sonra bir bosluk iyi gider classtan önce